admin-dev/themes/new-theme/js/pages/order/create.js

```diff
@@ -50,10 +50,21 @@ function refreshAddressesList(refreshCartAddresses) {
   }
 }
 
+/**
+ * proxy to allow other scripts within the Create Order page to refresh cart
+ */
+function refreshCart() {
+  if (orderPageManager === null) {
+    console.log('Error: Could not refresh addresses list as orderPageManager is null');
```

Comment:
You forgot `return`... else it will try `refreshCart` on `null`

Reply:
thank you, just added!